packages/create-redwood-app/template/README.md

```diff
@@ -1,21 +1,62 @@
-# Redwood
-## Getting Started
-- [Tutorial](https://redwoodjs.com/docs/tutorial): getting started and complete overview guide.
-- [Docs](https://redwoodjs.com/docs/introduction): using the Redwood Router, handling assets and files, list of command-line tools, and more.
-- [Redwood Community](https://community.redwoodjs.com): get help, share tips and tricks, and collaborate on everything about RedwoodJS.
+# README
```

Comment:
See my comments in the Quick-start.

Additionally, I think this should include a Section with something like "Quick Link": website, docs, twitter & newsletter, discord, discourse, etc.:
- how to stay updated: Forum Announcements and Twitter
- how to join the community
- how to submit bugs: GitHub Issues
- Contributors are welcome! Link to docs/contributing

Reply:
FYI: sometime in April I'm going to create a Email list people can sign up for to go through the tutorial in 8 weeks. Will make for a good next step for these types of things.